src/Features/Core/Portable/CodeFixes/PredefinedCodeFixProviderNames.cs

```diff
@@ -13,6 +13,7 @@ internal static class PredefinedCodeFixProviderNames
         public const string ConvertToAsync = "Convert To Async";
         public const string ConvertToIterator = "Convert To Iterator";
         public const string CorrectNextControlVariable = "Correct Next Control Variable";
+        public const string RemoveDuplicateParamTag = "Remove Duplicate Param Tag";
```

Comment:
I think you changed this name below.  It would be nice to keep this in sync with the feature name.